### PR TITLE
Support configure FFMPEG_BINARIES_URL 

### PR DESCRIPTION
**Pull request by chenpx976**
```
npm install ffmpeg-static --ffmpeg_binaries_url=https://cdn.npmmirror.com/binaries/ffmpeg-static

FFMPEG_BINARIES_URL=https://cdn.npmmirror.com/binaries/ffmpeg-static npm install ffmpeg-static
```

@derhuerst

### CODE REVIEW

**Comment by nononoyeah** [comment]
+1 agree

**Comment by derhuerst** [comment]
@chenpx976 We could also make the download URL customisable via an environment variable, e.g. `$FFMPEG_BINARIES_URL`.

> I have the same advice, affected by different regions, downloading with the default URL […] failed sometimes.

@nononoyeah I'm curious, do know in which specific cases it fails? Regardless of a customisable download URL, we should strive to pick a default host that is as accessible as possible!

**Comment by chenpx976** [comment]
@derhuerst  
I fixed environment variable  `FFMPEGSTATIC_CDNURL ` => `FFMPEG_BINARIES_URL `

File: README.md [hunk 1]
```diff
@@ -21,23 +21,23 @@ $ npm install ffmpeg-static
...
 
 Or add property into your [`.npmrc`](https://docs.npmjs.com/files/npmrc) file.
 
 ```
-ffmpegstatic_cdnurl=https://cdn.npmmirror.com/binaries/ffmpeg-static
+ffmpeg_binaries_url=https://cdn.npmmirror.com/binaries/ffmpeg-static
 ```
 
-Another option is to use PATH variable `FFMPEGSTATIC_CDNURL`.
+Another option is to use PATH variable `FFMPEG_BINARIES_URL`.
```
Comment (derhuerst) [inline README.md:37]:
```suggestion
Another option is to use the `FFMPEG_BINARIES_URL` environment variable.
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,27 @@ $ npm install ffmpeg-static
...
+Or add property into your [`.npmrc`](https://docs.npmjs.com/files/npmrc) file.
+
+```
+ffmpeg_binaries_url=https://cdn.npmmirror.com/binaries/ffmpeg-static
+```
+
+Another option is to use PATH variable `FFMPEG_BINARIES_URL`.
+
+```shell
+FFMPEG_BINARIES_URL=https://cdn.npmmirror.com/binaries/ffmpeg-static npm install ffmpeg-static
```
Comment (derhuerst) [inline README.md:40]: Just setting the variable won't export it, so it won't be accessible to `ffmpeg-static`'s `node install.js` (which is a child process of `npm install`).

```suggestion
env export FFMPEG_BINARIES_URL=https://cdn.npmmirror.com/binaries/ffmpeg-static npm install ffmpeg-static
```

**Comment by derhuerst** [comment]
I have though about supporting the npm config variable and decided to remove it:

- Using normal environment variables should work just fine, the npm config way barely provides additional convenience.
- The npm config mechanism is npm-specific; While Yarn supports it for backwards-compatibility, other package managers might not.

**Comment by nononoyeah** [comment]
> @chenpx976 We could also make the download URL customisable via an environment variable, e.g. `$FFMPEG_BINARIES_URL`.
> 
> > I have the same advice, affected by different regions, downloading with the default URL […] failed sometimes.
> 
> @nononoyeah I'm curious, do know in which specific cases it fails? Regardless of a customisable download URL, we should strive to pick a default host that is as accessible as possible!

like this 


> @chenpx976 We could also make the download URL customisable via an environment variable, e.g. `$FFMPEG_BINARIES_URL`.
> 
> > I have the same advice, affected by different regions, downloading with the default URL […] failed sometimes.
> 
> @nononoyeah I'm curious, do know in which specific cases it fails? Regardless of a customisable download URL, we should strive to pick a default host that is as accessible as possible!

like this
image

**Comment by chenpx976** [comment]
The network environment for accessing github in China is very bad, so use [cnpm](https://github.com/cnpm/cnpm) instead of npm
the mirror config is https://github.com/cnpm/binary-mirror-config/blob/master/package.json#L47
I will add an environment variable to the mirror config

**Comment by derhuerst** [comment]
Thanks. Do you want to be mentioned as a [contributor](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors)? If yes, with which string?

**Comment by derhuerst** [comment]
Published as `ffmpeg-static@5.1.0`. 🚀

**Pull request closed without merging**